src/viur/core/prototypes/list.py

```diff
@@ -569,6 +636,36 @@ def onDeleted(self, skel: SkeletonInstance):
         if user := current.user.get():
             logging.info(f"""User: {user["name"]!r} ({user["key"]!r})""")
 
+    def onClone(self, skel: SkeletonInstance, src_skel: SkeletonInstance):
+        """
+            Hook function that is called before cloning an entry.
+
+            It can be overwritten to a module-specific behavior.
+
+            :param skel: The new Skeleton that is being created.
+            :param src_skel: The source Skeleton `skel` is cloned from.
+
+            .. seealso:: :func:`clone`, :func:`onCloned`
+        """
+        pass
+
+    def onCloned(self, skel: SkeletonInstance, src_skel: SkeletonInstance):
+        """
+            Hook function that is called after cloning an entry.
+
+            It can be overwritten to a module-specific behavior.
+
+            :param skel: The new Skeleton that was created.
+            :param src_skel: The source Skeleton `skel` was cloned from.
+
+            .. seealso:: :func:`clone`, :func:`onClone`
```

Comment:
```suggestion
        Hook function that is called after cloning an entry.

        It can be overwritten to a module-specific behavior.

        :param skel: The new SkeletonInstance that was created.
        :param src_skel: The source SkeletonInstance `skel` was cloned from.

        .. seealso:: :func:`clone`, :func:`onClone`
```